docs/api/web-contents.md

```diff
@@ -369,6 +369,14 @@ Emitted when the window enters a full-screen state triggered by HTML API.
 
 Emitted when the window leaves a full-screen state triggered by HTML API.
 
+#### Event: 'zoom-changed'
+
+Returns:
+* `event` Event
+* `zoomedIn` Boolean - true if the user is trying to zoom in, false if the user is trying to zoom out
```

Comment:
Is this really the Chromium API?

We must be able to do better than this.  How about making this `wheelDeltaY` similar to the JS side scroll event with `+1` and `-1` as possible values.  I won't block this PR but a boolean indicating zooming in / our seems like a bit of a rough API here

Reply:
@ppontes It makes sense

Reply:
@ppontes what do you think?

Reply:
@MarshallOfSound, I find it odd if an event called `zoom-changed` will have a `wheelDeltaY` parameter - they don't seem semantically related. `zoom-changed` represents a complete zoom action. Users will likely be interested in knowing if the zoom level is to increase or decrease, not in the particular movement of the mouse wheel (even though they're correlated). That's how I understood the Chromium API and kept it. But I don't oppose changing the parameter as long as we also rename the event so that its name makes it clear this is a CTRL + mouse wheel event. Something like `mouse-wheel-zoom-changed`. What do you think?

Reply:
Hm this is a good point, I guess what I'm trying to convey here is that using a Boolean to represent a direction feels wrong to me.

Maybe an enum `ZoomDirection.OUT`?